dask_cloudprovider/providers/aws/ecs.py

```diff
@@ -1214,14 +1197,126 @@ class FargateCluster(ECSCluster):
     kwargs: dict
         Keyword arguments to be passed to :class:`ECSCluster`.
 
+    Examples
```

Comment:
The intention here is that this will be rendered in RTD, correct ?

Reply:
Yeah